release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1165,6 +1165,8 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-rhcos-bug-fixes"]
 ==== {op-system-first}
 
+* Previously, on Azure, the SR-IOV interface was configured by NetworkManager during boot because the udev rule that marks it as `NM_UNMANAGED` wasn't in the `initramfs` files. With this update, the udev rule is now in the `initramfs` files and the SR-IOV interface should always be unmanaged by NetworkManager. (link:https://issues.redhat.com/browse/OCPBUGS-7173[*OCPBUGS-7173*])
```

Comment:
```suggestion
* Previously, on Azure, the SR-IOV interface was configured by NetworkManager during boot because the udev rule that marks it as `NM_UNMANAGED` wasn't in the `initramfs` file. With this update, the udev rule is now in the `initramfs` file and the SR-IOV interface should always be unmanaged by NetworkManager. (link:https://issues.redhat.com/browse/OCPBUGS-7173[*OCPBUGS-7173*])
```

Reply:
Thanks, Jesse. That `s` tricked me twice haha.